app/uk/gov/hmrc/selfservicetimetopay/config/frontendWiring.scala

```diff
@@ -101,10 +101,9 @@ trait TimeToPayController extends FrontendController with Actions with CheckSess
   def AuthorisedSaUser(body: AsyncPlayUserRequest): PlayAction[AnyContent] = AuthorisedFor(SaRegime, GGConfidence).async(body)
 
   override implicit def hc(implicit request: Request[_]): HeaderCarrier = {
-    super.hc(request).withExtraHeaders(ttpSessionId -> "123")
-//    request.session.get(ttpSessionId).fold(super.hc(request)) { id =>
-//      super.hc(request).withExtraHeaders(ttpSessionId -> id)
-//    }
+    request.cookies.find(_.name == ttpSessionId).fold(super.hc(request)) { id =>
```

Comment:
If ttpCookie is present, map the value in to the HeaderCarrier's extra parameters.